src/dex-ui/pages/dao/DAODashboard.tsx

```diff
@@ -0,0 +1,181 @@
+  const navigate = useNavigate();
+  const { accountId: daoAccountId = "" } = useParams();
+
+  const location = useLocation();
+
+  const currentTabNameByRoute = location.pathname.split("/").at(-1) ?? "";
+  const tabIndexByRoute = daoNavigationTabs.map((tab) => tab.title.toLowerCase()).indexOf(currentTabNameByRoute);
+  const intialTabIndex = tabIndexByRoute === -1 ? 0 : tabIndexByRoute;
+  const { handleTabChange } = useTabFilters(intialTabIndex);
+
+  const notification = useNotification({
+    successMessage: location.state?.createDAOSuccessMessage,
+    transactionState: location.state?.transactionState,
+  });
+
+  function onBackToDAOsLinkClick() {
+    navigate(Paths.DAOs.absolute);
+  }
+
+  if (isError) {
+    return <ErrorLayout message={errorMessage} />;
+  }
+
+  if (isLoading) {
+    return <LoadingSpinnerLayout />;
+  }
+
+  if (isNotFound) {
+    return (
+      <NotFound
+        message={`We didn't find any data for this DAO (${daoAccountId}).`}
+        preLinkText={""}
+        linkText={"Click here to return to the DAOs list page."}
+        onLinkClick={onBackToDAOsLinkClick}
+      />
+    );
+  }
+
+  if (isDAOFound && dao?.isPrivate) {
+    return (
+      <NotFound
+        message={`This DAO is private (${daoAccountId}).`}
+        preLinkText={""}
+        linkText={"Click here to return to the DAOs list page."}
+        onLinkClick={onBackToDAOsLinkClick}
+      />
+    );
+  }
+
+  if (dao && isDAOFound && isSuccess) {
+    const { accountId, type, name } = dao;
+
+    return (
+      <Page
+        gap={0}
+        type={PageLayout.Dashboard}
+        header={
+          <>
+            {notification.isSuccessNotificationVisible && (
+              <Center>
+                <Box padding="16px 80px 0 80px" maxWidth="fit-content" paddingTop="1rem">
+                  <Notification
+                    type={NotficationTypes.SUCCESS}
+                    textStyle="b3"
+                    message={notification.successNotificationMessage}
+                    isLinkShown={true}
+                    linkText="View in HashScan"
+                    linkRef={notification.hashscanTransactionLink}
+                    isCloseButtonShown={true}
+                    handleClickClose={notification.handleCloseNotificationButtonClicked}
+                  />
+                </Box>
+              </Center>
+            )}
+            <DashboardHeader daoAccountId={accountId} name={name} type={type} />
+          </>
+        }
+        body={
+          <Tabs defaultIndex={intialTabIndex} onChange={handleTabChange} isLazy bg={Color.White_02}>
+            <Flex flex="row" padding="0px 80px" borderBottom={`1px solid ${Color.Neutral._200}`}>
+              <TabList borderBottom="0">
+                {daoNavigationTabs.map((tab, index: number) => {
+                  return (
+                    <Tab
+                      key={index}
+                      tabIndex={index}
+                      height="fit-content"
```

Comment:
Move these styles under a variant in the `TabsStyles.ts` file. You can edit the existing `tabList` and `tab` variants - I don't believe they're being used anywhere.

Reply:
The general approach for styling is to either create a ChakraUI style variant if the styles apply to a reusable component or apply the styles inline if the style is very specific to a non-reusable element on a page.